source/agora/consensus/Reward.d

```diff
@@ -0,0 +1,410 @@
+/*******************************************************************************
+
+    Contains functions to calculate the block reward.
+
+    Block reward are given to validators for verifying transactions and signing
+    blocks. This serves as an incentive for coin holders to participate in the
+    network as validators. The more active validators the network has, the more
+    signatures are on the blocks, and ultimately the more secure the overall
+    network is.
+
+    Block reward is also sent to the Commons Budget address for the first
+    6.7 years. After that period, the Commons Budget projects will be funded
+    from the transaction fees.
+
+    Here are some finer details of the block reward calculation
+
+    $(UL
+
+    $(LI Block rewards will be distributed in every Coinbase payout block which
+    is also used to payout the transaction fees and data fees.)
```

Comment:
"Every block that includes a coinbase transaction" ?
Which is also used to ~payout~ => distribute ?